src/providers/MeetingProvider/docs/MeetingManager.stories.mdx

```diff
@@ -108,3 +112,43 @@ const MyApp = () => {
   }, [])
 }
 ```
+
+2. Share the same `MeetingManager` instance with multiple different `MeetingProvider`s.
+
+You can create a new `MeetingManager` instance and then pass it as a prop to your `MeetingProvider`s. If not passed,
+a new `MeetingManager` instance will be created internally with each `MeetingProvider` when rendered; and, 
+you will get the `MeetingManager` instance associated with that particular `MeetingProvider` when `useMeetingManager` hook is called.
```

Comment:
We should very explicitly call out that this is to support unique cases where you may have different React trees within your applications (such as when migrating from an Angular application to React)

Reply:
Yup, thanks, good point. I have added this in at the start of the `MeetingManager` documentation.